schemas.go

```diff
@@ -23,9 +23,9 @@ const (
 	CustomXMLTypeStrict          = "http://purl.oclc.org/ooxml/officeDocument/relationships/customXml"
 
 	// SML strict
-	WorksheetTypeStrict    = "http://purl.oclc.org/ooxml/officeDocument/relationships/worksheet"
-	SharedStingsTypeStrict = "http://purl.oclc.org/ooxml/officeDocument/relationships/sharedStrings"
-	TableTypeStrict        = "http://purl.oclc.org/ooxml/officeDocument/relationships/table"
+	WorksheetTypeStrict     = "http://purl.oclc.org/ooxml/officeDocument/relationships/worksheet"
+	SharedStringsTypeStrict = "http://purl.oclc.org/ooxml/officeDocument/relationships/sharedStrings"
```

Comment:
To keep compatibility, can you do
SharedStingsTypeStrict = SharedStringsTypeStrict
with a comment // Deprecated: Renamed to SharedStringsTypeStrict, will be removed in next version.
Also for 
SharedStingsType = SharedStringsType  below.

Want to try to keep our compatibility as good as possible.  It's not fun getting compile errors in dependencies.

Reply:
done